snippets/espressif-mcuboot/esp-mcuboot.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BOOTLOADER_MCUBOOT=y
```

Comment:
Why is this a snippet ?

Please explain why `-DSNIPPET=esp-mcuboot` is cleaner, better, easier than `-DCONFIG_BOOTLOADER_MCUBOOT=y`

Reply:
Yeah, the idea here is to use them in combination. I'm using `-DSNIPPET="mcuboot;spiram"`, which is much shorter compared to adding `-DCONFIG_BOOTLOADER_MCUBOOT=y`

Reply:
> Why is this a snippet ?
> 
> Please explain why `-DSNIPPET=esp-mcuboot` is cleaner, better, easier than `-DCONFIG_BOOTLOADER_MCUBOOT=y`

We often need to switch between no-bootloader (simpleboot) to MCUBoot. And this - yes very primitive - mcuboot snippet allows us to use snippets in a linear fashion:
`west build -p -b <board> <sample> -DSNIPPER="mcuboot;spiram;..."`